api/avatar/avatarRouter.js

```diff
@@ -1,8 +1,9 @@
 const router = require('express').Router();
-const authRequired = require('../middleware/authRequired');
+
+const { authRequired, avatarValidation, fileUpload } = require('../middleware');
+const { ops } = require('../../lib');
```

Comment:
what is this?

Reply:
I changed the middleware to be a module-based export from the index file! And the ops module has functions for basic CRUD operations on data

Reply:
This is such a cool idea

Reply:
![wow](https://user-images.githubusercontent.com/59833966/95139423-09da6900-0732-11eb-8e4b-27ea544a52e4.jpeg)